src/components/MarkdownEditor/EditorToolbar.tsx

```diff
@@ -0,0 +1,312 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { useCallback, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  $getSelection,
+  $isRangeSelection,
+  $isRootOrShadowRoot,
+  ElementNode,
+  RangeSelection,
+  TextNode,
+  FORMAT_TEXT_COMMAND,
+  SELECTION_CHANGE_COMMAND,
+  COMMAND_PRIORITY_CRITICAL,
+  COMMAND_PRIORITY_NORMAL,
+  KEY_MODIFIER_COMMAND,
+} from 'lexical';
+import styled from '@emotion/styled';
+import { $isAtNodeEnd } from '@lexical/selection';
+import { $isLinkNode, TOGGLE_LINK_COMMAND } from '@lexical/link';
+import {
+  $isListNode,
+  INSERT_ORDERED_LIST_COMMAND,
+  INSERT_UNORDERED_LIST_COMMAND,
+  ListNode,
+  REMOVE_LIST_COMMAND,
+} from '@lexical/list';
+import {
+  $findMatchingParent,
+  mergeRegister,
+  $getNearestNodeOfType,
+} from '@lexical/utils';
+import { useLexicalComposerContext } from '@lexical/react/LexicalComposerContext';
+import { IconButtonV2 } from '@ndla/button';
+import { colors, misc, spacing } from '@ndla/core';
+import {
+  Bold,
+  Italic,
+  Link,
+  ListCircle,
+  ListNumbered,
+} from '@ndla/icons/editor';
+import { ADD_LINK_COMMAND } from './FloatingLinkEditorPlugin';
+
+const osCtrl = (key: string) => {
+  const ua =
+    typeof navigator !== 'undefined' ? navigator?.userAgent : undefined;
+  if (ua?.includes('Macintosh')) {
+    return `⌘+${key}`;
+  } else if (ua?.includes('Linux') || ua?.includes('Windows')) {
```

Comment:
```suggestion
  } else if (ua?.includes('Linux') || ua?.includes('Windows') || ua?.includes('CrOS') {
```
ChromeOS bruker Ctrl på samme måte som Windows, iallefall for Ctrl+B, Ctrl+I og Ctrl+U, såvidt jeg kunne finne ut